internal/pkg/config/loader_test.go

```diff
@@ -133,68 +106,45 @@ func TestMemoryLoaderImplementsLoader(t *testing.T) {
 func TestMemoryLoaderReadEmptyConfig(t *testing.T) {
 	// First read without a write should return an error.
 	loader, err := NewMemoryLoader(nil)
-	if err != nil {
-		t.Fatalf("NewMemoryLoader() returned error: %v", err)
-	}
+	require.NoError(t, err, err)
 
 	gotExists, err := loader.Exists()
-	if err != nil {
-		t.Errorf("MemoryLoader.Exists() error = %v", err)
-	} else if gotExists {
-		t.Errorf("MemoryLoader.Exists() returned true")
-	}
+	require.NoError(t, err, err)
+	assert.False(t, gotExists, "MemoryLoader.Exists() returned true")
 
 	_, gotErr := loader.Read()
-	if gotErr == nil {
-		t.Fatal("MemoryLoader.Read() did not return error")
-	}
+	require.Error(t, gotErr, gotErr)
```

Comment:
Is this check correct?

Reply:
Yes - previous code failed if there was no error.